packages/components/src/playground/button.tsx

```diff
@@ -20,13 +20,7 @@ const ButtonComponent = styled.button<BaseButtonProps>(({ variant }) => [
 function Button(props: ButtonProps): JSX.Element {
   const { children, ...rest } = props;
 
-  return (
-    <ButtonComponent {...rest}>
-      <Typography bold color="white" size="button">
-        {children}
-      </Typography>
-    </ButtonComponent>
-  );
+  return <ButtonComponent {...rest}>{children}</ButtonComponent>;
```

Comment:
doesn't matter that much since it's a playground, but following up on a previous comment that this probably shouldn't use Typography

Reply:
Oh yes, button should definitely not use typography (no need to nest a p in a button)